### PR TITLE
Document how to use tenant release pipelines

### PR DESCRIPTION
**Pull request by ralphbean**
I haven't had a chance to test this functionally. If you get a chance to try it out, please provide feedback here so we can clean it up and tune it.

### CODE REVIEW

File: docs/modules/ROOT/pages/advanced-how-tos/releasing/tenant-release-pipelines.adoc [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
++
+[source,yaml]
+----
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleasePlan
+metadata:
+ labels:
+   release.appstudio.openshift.io/auto-release: 'true' <.>
+   release.appstudio.openshift.io/standing-attribution: 'true'
```
Comment (arewm) [inline docs/modules/ROOT/pages/advanced-how-tos/releasing/tenant-release-pipelines.adoc:32]: Is this still required for just a tenant release pipeline?

File: docs/modules/ROOT/pages/advanced-how-tos/releasing/tenant-release-pipelines.adoc [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+[source,shell]
+----
+kubectl apply -f ReleasePlan.yaml -n dev
+----
+
+. Provision any secrets needed by the particular release pipeline you chose and xref:/how-tos/configuring/creating-secrets.adoc[upload them to the Development workspace].
+
+.Next steps
+
+* *Create a `release` object:* The development team creates a Release object to references a specific Snapshot and ReleasePlan. It indicates the users' intent to release that Snapshot via the tenant release pipeline defined in the ReleasePlan.
```
Comment (arewm) [inline docs/modules/ROOT/pages/advanced-how-tos/releasing/tenant-release-pipelines.adoc:79]: Is the UI currently limited in that it doesn't show these pipelines?

**Review by arewm — COMMENTED** [review]
Does it make sense to add a blurb about these to https://konflux-ci.dev/docs/advanced-how-tos/releasing/ as well?

File: docs/modules/ROOT/pages/advanced-how-tos/releasing/tenant-release-pipelines.adoc (hunk 2 above)
Comment (mmalina) [reply docs/modules/ROOT/pages/advanced-how-tos/releasing/tenant-release-pipelines.adoc:79]: Yes, that's correct. @davidmogar was asking the UI team on how to proceed, but I'm not sure what the outcome of the discussion was (hopefully a jira).

Comment (davidmogar) [reply docs/modules/ROOT/pages/advanced-how-tos/releasing/tenant-release-pipelines.adoc:79]: I'm in talks with the UX team and we'll schedule a meeting to address the lack of support for showing our pipelines. There was an initial design to show release pipelines in general and not just the managed one, but no progress was done. Now that we are adding the final pipeline next week, we really need to show this info.

**Review by arewm — APPROVED** [review]
Let's get this PR merged. I think the content is good. If it needs to be improved, we can iterate.

File: docs/modules/ROOT/pages/advanced-how-tos/releasing/tenant-release-pipelines.adoc (hunk 1 above)
Comment (arewm) [reply docs/modules/ROOT/pages/advanced-how-tos/releasing/tenant-release-pipelines.adoc:32]: I checked with @theflockers. This is still required. If it moves to be optional, we can update the docs.

Comment (johnbieren) [reply docs/modules/ROOT/pages/advanced-how-tos/releasing/tenant-release-pipelines.adoc:32]: The release will fail validation (before any pipelines at all run) if the attribution data is not valid. This is part of that

File: docs/modules/ROOT/pages/advanced-how-tos/releasing/tenant-release-pipelines.adoc [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+ data:
+   mapping:
+     components: <.>
+       - name: demo-component-1
+         repository: registry/destination-image-repository-1
+         tags: [latest]
+       - name: demo-component-2
+         repository: registry/destination-image-repository-2
+         tags: [latest]
+ pipelineRef: <.>
```
Comment (johnbieren) [inline docs/modules/ROOT/pages/advanced-how-tos/releasing/tenant-release-pipelines.adoc:46]: @seanconroy2021 when your final pipeline stuff merges, can you update this doc? I believe you renamed the field (bonus points if you can also fix the typo on line 55 at the same time)

**Pull request merged**